tools/generator/dfg/stm32/stm.py

```diff
@@ -6,6 +6,18 @@
 
 LOGGER = logging.getLogger("dfg.stm.data")
 
+ignored_devices = \
+[
+    "STM32U59",
+    "STM32U5A",
+]
+
+def ignoreDevice(device_id: str) -> bool:
+    for ignore in ignored_devices:
+        if device_id.startswith(ignore):
+            return True
+    return False
```

Comment:
Not sure if that is the best way to ignore some devices (here: STM32U5{95, 99, A5, A9}) that are not published (yet) and thus are missing datasheet and relevant information.

I my opinion I it not useful to generate device files for this devices so we should somehow ignore them.

Reply:
Meh, doesn't look like we have a choice if the documents are not there. We could guess the memory layout, but then it'll just be a problem later.